ConsoleApplication4.cpp

```diff
@@ -0,0 +1,119 @@
+﻿#include <iostream>
+#include <string>
+using namespace std;
+
+class Fish{
+public:
+  int age;
+  int neededSpace;
+  int size;
+  int freeSpace = 22;
+  string name;
+  string species;
+  string food;
+  bool isAggressive;
+
+  void Info(){
```

Comment:
There should be a space between the close parenthesis and the open curly brace. (So everywhere)